eks/verify_kubera_on_eks.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+ 
+set +e
+
+## Creating AWS cluster with two nodes and noty type t2.xlarge
+## in region us-east-2
+eksctl create cluster --name=kubera-cluster --node-type=t2.xlarge
+kubectl get nodes -owide
+sleep 200
+kubectl get pod -A
+
+## Install helm
+curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3
+chmod 700 get_helm.sh
+./get_helm.sh
+helm version
+
+## Creates secret this secret will help us to pull images from registry
+kubectl create secret docker-registry directoronprem-registry-secret \
+--docker-server=registry.mayadata.io \
+--docker-username="$DOCKER_USER" --docker-password="$DOCKER_PASS"
+kubectl get secret
+
+## To access kubera installation approved solution is to use an external Load Balancer 
+## with DNS to point to the hostname you are using for server.url
+IP=$(sudo kubectl get nodes -o wide --no-headers | awk '{print $6}' | head -n 1)
+URL=http://$IP
+helm repo add kubera https://charts.mayadata.io/
+helm repo update
+helm install kubera kubera/kubera-charts --set server.url="$URL"
+sleep 300
```

Comment:
Can we avoid sleep

Reply:
No, we can't because we need sleep so that all the pods should come in `Running` state @AmitKumarDas . We can remove this sleep once we have sanity check for all the kubera components.